homeassistant/components/met/weather.py

```diff
@@ -111,7 +112,7 @@ def name(self) -> str:
         name = self._config.get(CONF_NAME)
         name_appendix = ""
         if self._hourly:
-            name_appendix = " Hourly"
+            name_appendix = " hourly"
 
         if name is not None:
             return f"{name}{name_appendix}"
```

Comment:
Hm I merged this but what is `name` ? Isn't that going to be the service name and we should never prefix it? Same with `DEFAULT_NAME`?

Reply:
Hmm seems to be some old stuff from the YAML import days.
It seems like the old `name` configured in YAML was stored in the config entry at the time (when imported). And still taken into consideration.